libs/ptformat/wscript

```diff
@@ -38,7 +38,7 @@ def build(bld):
         obj.cflags = [ bld.env['compiler_flags_dict']['pic'] ]
         
     obj.export_includes     = ['.']
-    obj.includes     = ['.']
+    obj.includes = '. /usr/include/glib-2.0 /usr/lib/glib-2.0/include'
```

Comment:
Ditto. Another absolute path, Prefer pkg-config or obj.uselib

Reply:
`obj.uselib` does not work here.
I've now pushed the use of `bld.env['INCLUDES_GLIB']`, but lists may not be added to the includes list like that. Will have to use a for loop